views/SignedInView.tsx

```diff
@@ -9,24 +9,23 @@ import {
   useColorModeValue,
 } from "@chakra-ui/react";
 import truncate from "smart-truncate";
-import { useManageSubscription, useW3iAccount } from "@web3inbox/widget-react";
+import { useWeb3InboxAccount, useSubscription } from "@web3inbox/react";
 
 import PushSubscription from "../components/PushSubscription";
 import CopyIcon from "../components/core/CopyIcon";
 import GmCard from "../components/core/GmCard";
 import Zorb from "../components/core/Zorb";
 import useThemeColor from "../styles/useThemeColors";
 import { useAccount } from "wagmi";
+import DevTimeStamp from "../components/DevTimeStamp";
 
 const SignedInView: React.FC = () => {
-  const { account, setAccount } = useW3iAccount();
-  const { isSubscribed } = useManageSubscription(account);
-  const { address } = useAccount({
-    onDisconnect: () => {
-      setAccount("");
-      window.location.reload();
-    },
-  });
+  const { data: subscription } = useSubscription();
+  const isSubscribed = Boolean(subscription)
+
+  const { address } = useAccount();
+
+  useWeb3InboxAccount(`eip155:1:${address}`);
```

Comment:
Maybe not now but I think we could find better way to handle this too. We're using template literals in several places to create CAIP-10 string, instead, it would be greate to automate it as well, or providing a util function to user to create it insated of template literals, somehow it bothers me

Reply:
Hm. If you were using w3i as a dev, would you prefer

```typescript
  useWeb3InboxAccount(`eip155:1:${address}`);
  //or
  useWeb3InboxAccount(getCaip10(chain, addres));
  //or
  // assume you have wagmi and use that under the hood to automate everything, no hook needed

Reply:
> providing a util function to user to create it insated of template literals

Technically all the fields would need options when we support more chains, so it would like `getCaip10("eip155", 1, address)` which is almost redundant. Also the format differs between chains, so Solana might not even have 3 fields like this